sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -327,14 +355,15 @@ export class MessageSender extends LinkEntity {
         }
       });
 
+    const retryOptions = options.retryOptions || {};
+    retryOptions.maxRetries = retryOptions.maxRetries ?? Constants.defaultMaxRetries;
+    retryOptions.retryDelayInMs =
+      retryOptions.retryDelayInMs ?? Constants.defaultDelayBetweenOperationRetriesInMs;
```

Comment:
Can we not skip the above checks for `maxRetries` and `retryDelayInMs` as the same will be done by the `retry()` method anyway?

Reply:
Makes sense.. removed.